src/pages/Explore/TraceExploration.tsx

```diff
@@ -204,40 +165,22 @@ export class TraceExploration extends SceneObjectBase<TraceExplorationState> {
     return this.getMetricVariable().getValue() as MetricFunction;
   }
 
+  public closeDrawer() {
+    this.setState({ traceId: undefined, spanId: undefined });
+  }
+
   static Component = ({ model }: SceneComponentProps<TraceExploration>) => {
-    const { singleBody, splitBody } = model.useState();
+    const { body } = model.useState();
     const styles = useStyles2(getStyles);
 
-    // The API looks a bit weird because duh we are opened if we are here, but this specifically means we are in a
-    // sidecar with some other app. In that case we don't want to show additional split layout as there is not much
-    // space and 3 splits is a bit too much.
-    const body: SceneObject =
-      config.featureToggles.appSidecar && sidecarServiceSingleton_EXPERIMENTAL?.isAppOpened(pluginJson.id)
-        ? singleBody
-        : splitBody;
-
     return <div className={styles.bodyContainer}> {body && <body.Component model={body} />} </div>;
   };
-
-  // Because we have two modes of how to present the secondary scene depending on sidecar we need to set it 2 ways.
-  // This could have been easier but we have to keep the split scene in memory to no trigger unmount (this is a bit
-  // different from react where components can be returned from a function without triggering unmount) and use
-  // `setState` to change the secondary scene. We could also just keep the splitScene and not use the secondary prop
-  // for the sidecar use case but splitScene adds some css that makes responsive sizing in sidecar complicated.
-  private setSecondaryScene(scene?: SceneObject) {
-    this.state.splitBody.setState({ secondary: scene });
-    if (scene) {
-      this.setState({ singleBody: scene });
-    } else {
-      this.setState({ singleBody: new TraceExplorationScene({}) });
-    }
-  }
```

Comment:
Andrej had added this with a PR that tested opening sidecar from a button in Explore Logs that opened a sidecar for Explore Traces. I did not test that removing this did not affect that so please double check if necessary!